### PR TITLE
Cleanup and initial run

### PR DESCRIPTION
**Pull request by r4mtor**
I tried to understand the installation instructions.
I had problems setting up the project item-wise and I also removed PEP08 Warnings in the python code and dependencies in the setup.py

### CODE REVIEW

**Comment by goto40** [comment]
Thank you very much for your pull request. I will check your request in detail the next days... Looks good.

**Comment by r4mtor** [comment]
Maybe add this command  `git clone https://github.com/goto40/itemlang --recurse-submodule` to the main [README.rst](https://github.com/goto40/itemlang/blob/master/README.rst). If you only clone the project, the system tests doens't work.

File: system_tests/run.sh [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```
Comment (goto40) [inline system_tests/run.sh]: What exactly is the reason for the env-command here?

**Comment by goto40** [comment]
Looks very good. I will merge the pull request now.

File: system_tests/run.sh (hunk 1 above)
Comment (r4mtor) [reply system_tests/run.sh]: I do the code inspection with the pycharm community edition. Here are the result in xml format.
[PyCharmInspection.zip](https://github.com/goto40/itemlang/files/1791067/PyCharmInspection.zip)

**Pull request merged**